tests/ctl/core/test_config.py

```diff
@@ -46,6 +46,34 @@ def test_get_deprecated_api_config(test_deprecated_config_path: str) -> None:
     assert config.database.test_db == "fidesctl_test_deprecated"
 
 
+@patch.dict(
+    os.environ,
+    {
+        "FIDESCTL__API__DATABASE_HOST": "test_host",
+        "FIDESCTL__API__DATABASE_NAME": "test_db_name",
+        "FIDESCTL__API__DATABASE_PASSWORD": "test_password",
+        "FIDESCTL__API__DATABASE_PORT": "1234",
+        "FIDESCTL__API__DATABASE_TEST_DATABASE_NAME": "test_test_db_name",
+        "FIDESCTL__API__DATABASE_USER": "phil_rules",
```

Comment:
👀

Full disclosure, I went straight to the unit tests, so I'm glad to see these here